handel/src/services/dynamodb/params-schema.json

```diff
@@ -0,0 +1,361 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "type": "object",
+    "properties": {
+        "type": {
+            "type": "string",
+            "description": "The service type (should always be 'dynamodb')",
+            "pattern": "^dynamodb$",
+            "errorMesssage": "The 'type' parameter must be 'dynamodb'"
+        },
+        "table_name": {
+            "type": "string",
+            "minLength": 3,
+            "maxLength": 255,
+            "pattern": "^[a-zA-Z0-9_\\-.]$",
+            "errorMessage": "The table_name parameter must be between 3 and 255 characters long and may only include alphanumeric characters, underscores (_), hyphens (-), and dots (.)"
+        },
+        "partition_key": {
+            "type": "object",
+            "properties": {
+                "name": {
+                    "type": "string",
+                    "errorMessage": "The 'name' parameter must be a string"
+                },
+                "type": {
+                    "type": "string",
+                    "description": "The 'type' parameter may only be 'String' or 'Number'",
+                    "pattern": "^String|Number$",
+                    "errorMessage": "The 'type' parameter must be 'String' or 'Number'"
+                }
+            },
+            "required": [
+                "name",
+                "type"
+            ],
+            "additionalProperties": false,
+            "errorMessage": {
+                "required": {
+                    "name": "The 'name' field in the 'partition_key' section is required",
+                    "type": "The 'type' field in the 'partition_key' section is required"
+                },
+                "additionalProperties": "Invalid/Unknown property specified in the 'partition_key' section"
+            }
+        },
+        "sort_key": {
+            "type": "object",
+            "properties": {
+                "name": {
+                    "type": "string",
+                    "errorMessage": "The 'name' parameter must be a string"
+                },
+                "type": {
+                    "type": "string",
+                    "description": "The 'type' parameter may only be 'String' or 'Number'",
+                    "pattern": "^String|Number$",
+                    "errorMessage": "The 'type' parameter must be 'String' or 'Number'"
+                }
+            },
+            "additionalProperties": false,
```

Comment:
We'll want to add the "required" section in here for the "name" and "type" parameters inside "sort_key".